src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -12,43 +14,73 @@ namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
     public class TagHelperExecutionContext
     {
         private readonly List<ITagHelper> _tagHelpers;
+        private readonly Func<Task> _executeChildContentAsync;
+        private readonly Action _startWritingScope;
+        private readonly Func<TextWriter> _endWritingScope;
+        private string _childContent;
+
+        /// <summary>
+        /// Internal for testing purposes only.
+        /// </summary>
+        internal TagHelperExecutionContext(string tagName)
+            : this(tagName,
+                   uniqueId: string.Empty,
```

Comment:
Tests expect a specific output and if it's a GUID then we can't validate output from objects that utilize the `TagHelperExecutionContext`